server/src/main/java/org/jboss/as/server/moduleservice/ExternalModuleService.java

```diff
@@ -63,15 +63,20 @@
      * @return true if valid, false otherwise
      */
     public boolean isValid(String externalModule) {
-        return new File(externalModule).exists();
+        File f = new File(externalModule);
+        return f.exists() && !f.isDirectory();
```

Comment:
AIUI this has the effect of preventing this new functionality being used to add directories via Class-Path entries in  MANIFEST.MF.

Which is fine; users doing that is not part of the scope of the RFE that is driving this.

If that is true then WFCORE-4375 is not a Feature Request; it is an Enhancement. There is no end user accessible feature here, just some internal changes to make things possible. The point matters because a Feature Request has different requirements from an Enhancement.

Reply:
Correct, the intention here was to protect the uses of directory absolute paths defined in manifest Class-Path attribute.

Before this change, if a user configures a directory in Class-Path attribute, then an exception was thrown. Now if a directory is used, we warn and ignore. I think that's valid change, I cannot think in something relying on this exception to work, so warning+ignore sounds enough. 

Based on the above, @bstansberry can we still convert it as an enhancement?

Reply:
@yersan I think that's fine. I removed the feature label.